libs/test/SalesforceAnalyticsTest/src/com/salesforce/androidsdk/analytics/security/EncryptorTest.java

```diff
@@ -139,6 +146,53 @@ public void testEncryptDecryptWithDifferentKeys() {
 		}
 	}
 
+	/**
+	 * Check cipher returned by Encryptor.getEncryptingCipher
+	 */
+	@Test
+	public void testGetEncryptingCipher()
+		throws InvalidAlgorithmParameterException, NoSuchAlgorithmException, InvalidKeyException {
+    	Cipher cipher = Encryptor.getEncryptingCipher(makeKey("my-key"));
+    	Assert.assertEquals("Wrong algorithm", "AES/CBC/PKCS5Padding", cipher.getAlgorithm());
+		Assert.assertEquals("Wrong iv length", 16, cipher.getIV().length);
+		Assert.assertEquals("Wrong mode", 16, cipher.getBlockSize());
+	}
```

Comment:
Can't find a way to see the mode the cipher was initialized with. But the third test below makes sure encryption/decryption work as expected.